woltka/file.py

```diff
@@ -249,17 +249,19 @@ def write_readmap(fh, rmap, namedic=None):
     namedic : dict, optional
         Taxon name dictionary.
     """
+    def sortkey(x): return -x[1], x[0]
     for read, taxa in rmap.items():
-        row = [namedic[read]] if namedic and read in namedic else [read]
+        row = [read]
         if isinstance(taxa, dict):
-            for taxon, count in taxa.items():
-                row.append(f'{taxon}:{count}')
+            for taxon, count in sorted(taxa.items(), key=sortkey):
```

Comment:
Can you put a comment explaining what `sortkey` does here?

Reply:
Good point. Comment added to line #252: "# sort subjects by count (high-to-low) then by alphabet".